src/pages/data-visualization/advanced-charts/index.mdx

```diff
@@ -17,97 +17,404 @@ Advanced charts are a powerful way to display complex data sets. While less intu
+
+<Row>
+<Column  colLg={4} colMd={4} colSm={4} >
+
+![Heat map hover detail](images/04a_adv_heatmap_detail_288.png)
+
+<Caption>Exploration of a domain axis hover behavior</Caption>
+
+</Column>
+
+<Column  colLg={4} colMd={4} colSm={4} >
+
+![Heat map interactive legend detail](images/04b_adv_heatmap_detail_288.png)
+
+<Caption>Exploration of an interactive legend</Caption>
+
+</Column>
+</Row>
+
+#### Divergent scales
+
+Diverging scales show color progression in two directions: dialing down the contrast of the first hue from one end to a neutral color at the midpoint, then increasing the contrast of the second hue to the other end of the scale. The neutral midpoint is often referred to as the "inflection point" (for example, a 0 value midway between -100 and +100).
+
+[Divergent scales](../data-visualization/color-palettes#diverging-palettes) require 3 colors for their correspondent values (min, max and inflection point) and Carbon has provided two approved palettes for divergent visualizations. Although the examples here use the "discrete" (stepped scale) provided by the Carbon palette, divergent heat maps often leverage "continous" values (gradients) to accomodate situations that require more than 10 values.
+
+#### Inflection points
+
+While min/max values can be automatically detected from data, inflection points can vary depending on data type. We default to zero in order to accomodate the most common cases and include a feature flag within the code to specify exceptions. The data for a divergent heat map does not need to be symmetrical on either side of the inflection point. For example, the data set could have a zero value between -20 and +100 as well.
+
+<Row>
+<Column  colLg={12} colMd={8} colSm={4} >
+
+![Divergent heat map](images/05_adv_heatmap_928.png)
+
+<Caption>
+  Example of a symmetrical divergent heat map with optional cell borders
+</Caption>
+
+</Column>
+</Row>
+
+## Network diagrams
+
+A network diagram is a way of visually representing network architecture. It maps out network structure with a variety of icons and connecting lines. It’s ideal for sharing the layout of a network because the visual presentation makes it easier for users to understand complex connections at a glance. These diagrams are especially useful for isolating problems or when designing a new system.
+
+The example below illustrates a swimlane graph with expandable nodes developed by the Security team. In products like QRadar these diagrams are often paired with side and filter panels. However, a simplified version of the component is available in the Carbon Add-ons repo for IBM designers to adapt.
+
+<InlineNotification>
+
+**Note:** Some of the components below are available to consume in the Carbon Add-ons repo. Please check with the CLX team on Slack (#clx-framework) to verify before creating your own components [Carbon Add-ons repository](https://github.com/carbon-design-system/carbon-charts).
```

Comment:
Should this link to the carbon charts repo? If so, should we say Carbon charts rather than Add-ons repo? It could lead to confusion since there are multiple add-on repos.

Reply:
Yes, will update that! Good catch!